cmake/tribits/python_utils/gitdist-setup.sh

```diff
@@ -40,7 +40,7 @@ SCRIPT_DIR=`echo $BASH_SOURCE | sed "s/\(.*\)\/.*\.sh/\1/g"`
 
 existing_gitdist=`which gitdist 2> /dev/null`
 if [[ "${existing_gitdist}" == "" ]] ; then
-  echo "Setting alias gitdist=${SCRIPT_DIR}/gitdist"
+  #echo "Setting alias gitdist=${SCRIPT_DIR}/gitdist"
```

Comment:
Remove?

Reply:
Addressed in commit https://github.com/TriBITSPub/TriBITS/commit/701f5a1cd0266ad853d45b286258ae7f6c724157